.github/workflows/subgraph.yml

```
@@ -0,0 +1,35 @@
+name: Subgraph
+
+on:
+  pull_request:
+    branches:    
+      - main
+
```

We should also filter on changes to the subgraph directory.
```suggestion

  push:
    paths:
      - 'subgraph/**'
```

---

I think my other command is relevant to this as well: https://github.com/fleekxyz/non-fungible-apps/pull/118#discussion_r1103052509

---

How so? Even if we do `main` this applies.  This is about filtering on only changes to this folder. So this wont run on changes to other folders. But perhaps to improve we can add contracts here too so that if there is a contract change that breaks subgraph we catch it. But regardless I think this is not affected by what we decide for the other comment.

---

Every release to `main` will most likely have a code change and that code change is going to result in a new ABI, a new deployed instance of the contract and it makes sense for subgraph unit tests to require changes as well. But if we want to add `develop`, then this comment to add the path makes a difference so that this github action does not run when there's no change.